xls/dslx/type_system/scoped_fn_stack_entry.cc

```diff
@@ -28,11 +28,6 @@ ScopedFnStackEntry::ScopedFnStackEntry(DeduceCtx* ctx, Module* module)
   ctx->AddFnStackEntry(FnStackEntry::MakeTop(module));
 }
 
-// Args:
```

Comment:
q: any reasons those get removed?

Reply:
They're in the .cc file comments should be on the declaration in the .h